SpreadsheetStreams/Code/Core/StreamWritePackage.cs

```diff
@@ -14,9 +14,9 @@ internal class PackageWriteStream : IDisposable
         private List<ContentType> _ContentTypes = new List<ContentType>();
         private Dictionary<string, List<Relationship>> _PartRelationships = new Dictionary<string, List<Relationship>>();
 
-        internal PackageWriteStream(Stream outputStream)
+        internal PackageWriteStream(Stream outputStream, bool leaveOpenAfterClose)
```

Comment:
Naming of the new parameter is inconsistent. Please keep it `leaveOpen`. That's how it's called in ZipArchive and it's very clear :-)